jobs/ci-kubernetes-e2e-gce-etcd3-release-1.5.env

```diff
@@ -0,0 +1,11 @@
+### job-env
+JENKINS_PUBLISHED_VERSION=ci/latest-1.5
+GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
+GINKGO_PARALLEL=y
+PROJECT=k8s-gce-etcd3-1-5
+TEST_ETCD_IMAGE="3.0.14"
+TEST_ETCD_VERSION="3.0.14"
+STORAGE_BACKEND="etcd3"
```

Comment:
Yeah - the fact that it was caught by test is great (and thanks for pointing that no quotes should be in this file).